src/components/movie-card/movie-card.js

```diff
@@ -1,22 +1,27 @@
-import { Card, CardContent, CardMedia, Typography, CardActionArea } from "@mui/material";
-import poster from "./poster.jpg";
+import { Card, CardContent, CardMedia, Typography, CardActionArea, Chip } from "@mui/material";
+import { img_300, notfound_300 } from "../../services/media-service";
 
-export default function ActionAreaCard() {
+export default function MovieCard({id, vote, title, alt, poster, genres}) {
   return (
-    <Card sx={{minWidth: 250, marginBottom: 4, backgroundColor: '#151515'}}>
+    <Card sx={{width: 270, marginBottom: 4,position: 'relative', backgroundColor: '#151515'}} key={id}>
+      <Chip 
+        label={vote.toFixed(1)} 
+        color={vote >= 7 ? 'success' : vote < 4 ? 'error' : 'warning'}
```

Comment:
it's hard to read nested ternary operator, try to avoid this, it's not a good practice 
for example
```js
useMemo(() => {
  if (vote >= 7) return 'success';
  ...
  return 'warning'
}, [vote])
```

Reply:
Fixed!